core/eolearn/core/eoexecution.py

```diff
@@ -75,12 +75,15 @@ def _parse_execution_args(execution_args):
 
         return [EOWorkflow.parse_input_args(input_args) for input_args in execution_args]
 
-    def run(self, workers=1):
+    def run(self, workers=1, multiprocess=True):
         """ Runs the executor with n workers.
 
-        :param workers: Number of parallel processes used in the execution. Default is a single process. If set to
-            `None` the number of workers will be the number of processors of the system.
+        :param workers: Maximum number of workflows which will be executed in parallel. Default is a single workflow.
+            If set to `None` the number of workers will be the number of processors of the system.
         :type workers: int or None
+        :param multiprocess: If `True` it will use multiple processors for execution. If `False` it will run on
+            multiple threads of a single processor.
+        :type multiprocess: bool
```

Comment:
What about the fact that we don't really see this happening when we use the threadpoolexecutor?

Reply:
You are right, this docs string is not clear enough. Added a better explanation.